static/_redirects

```diff
@@ -360,6 +360,9 @@
 /ref/python/inittorch.view_as_real/ /ref/python/init/ 301
 /ref/python/integrations/sklearn/ /guides/integrations/scikit/ 301
 /ref/python/join/ /ref/python/ 301
+/ref/python/public-api/queuedrun/ /ref/python/init/ 301
```

Comment:
When I look at https://redirect-launch-apis-public.docodile.pages.dev/ref/python/init/, I see no reference to "launch", so if I actually encountered one of these redirects as a reader, I would be very confused. Is there anything that can be added to the target page to help?